dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+===== Other Details
+
+Votes are published on the Bitcoin network and _encrypted_ vote data objects are broadcast on the Bisq P2P network (hence the term 'blind' vote). The Bitcoin transaction includes a hash of the vote data object. BSQ specified for voting stake is also specified in this transaction, and cannot be spent until the vote reveal transaction is published.
+
+Following the end of the blind voting phase, there is a break of 10 blocks to protect against reorgs.
+
+==== Vote Reveal Phase
+
+_450 blocks, about 3 days_
+
+No action is required of the user during this phase aside from having the Bisq software online at some point during the phase so the vote reveal phase so it can publish the vote reveal transaction.
+
+You should see a pop-up in the UI when the transaction has been published.
+
+===== Other Details
+
+Only a Bitcoin transaction is made in this phase—no additional data is broadcast to the Bisq P2P network.
+
+This transaction includes 2 items:
+
+1. the key to decrypt the votes cast in the blind voting phase
+2. a hash of the list of voting data objects received by the Bisq node making the transaction (ordered by transaction ID; details <<#determining-consensus,below>>)
+
+One of the inputs to this transaction is the BSQ reserved for stake, so once this transaction is published, that BSQ becomes spendable again.
+
+If a user fails to make their vote reveal transaction, their vote is rendered invalid.
+
+Again, for protection against reorgs, there is a break of 10 blocks after this phase is over.
+
+==== Vote Result Phase
+
+_10 blocks, about 100 minutes_
+
+This phase is more of a trigger, so allotting 10 blocks is more for consistency than a hard requirement.
+
+In this phase, Bisq nodes calculate vote results from proposals, voting stake, and votes (which can now be decrypted from the vote reveal transactions).
+
+===== Determining Consensus
+
+We can trust that all Bisq nodes will get the same data from the the Bitcoin blockchain. However, we cannot be sure that all Bisq nodes have obtained identical data from the Bisq P2P network. It is critical that any potential inconsistencies be handled before determining vote results, otherwise, vote results may differ.
+
+This is why each Bisq node includes a hash of the list of the voting data objects it has received in its vote reveal transaction. While a hash of vote data objects is recorded on the Bitcoin blockchain for integrity, it's not guaranteed that every _voting data object itself_ is actually delivered to every Bisq node.
+
+So, when calculating vote results, the majority hash from the vote reveal transactions (majority by voting stake) is considered to be the authoritative representation of votes for the whole network. Vote result calculations are based on this uniform determination, ensuring consensus across the network.
```

Comment:
Maybe add: The majority hash need to have at least 80% of the total vote stake otherwise the whole voting cycle is considered invalid and all proposals are rejected. Inconsitency should be exceptional and if there is more then 20% of the network inconsistant it shows either a big problem with the P2P network stability. Beside that the 80% requirement protects against potential attacks as only an attacker with 80% of total stake could succeed.